VAMobile/src/screens/HomeScreen/VeteranStatusScreen/VeteranStatusScreen.tsx

```diff
@@ -6,30 +6,31 @@ import React, { FC } from 'react'
 
 import { BackgroundVariant, BorderColorVariant, Box, BoxProps, ClickToCallPhoneNumber, LargePanel, TextView, VAIcon } from 'components'
 import { BranchesOfServiceConstants, ServiceData } from 'store/api/types'
-import { DisabilityRatingState, MilitaryServiceState, PersonalInformationState } from 'store/slices'
+import { DisabilityRatingState, MilitaryServiceState } from 'store/slices'
 import { HomeStackParamList } from 'screens/HomeScreen/HomeStackScreens'
 import { NAMESPACE } from 'constants/namespaces'
 import { RootState } from 'store'
 import { displayedTextPhoneNumber } from '../../../utils/formattingUtils'
-import { getBirthDate } from 'screens/HomeScreen/ProfileScreen/PersonalInformationScreen/PersonalInformationScreen'
 import { useAuthorizedServices } from 'api/authorizedServices/getAuthorizedServices'
+import { usePersonalInformation } from 'api/personalInformation/getPersonalInformation'
 import { useTheme } from 'utils/hooks'
 // import PhotoUpload from 'components/PhotoUpload'
 
 type VeteranStatusScreenProps = StackScreenProps<HomeStackParamList, 'VeteranStatus'>
 
 const VeteranStatusScreen: FC<VeteranStatusScreenProps> = () => {
-  const { profile } = useSelector<RootState, PersonalInformationState>((state) => state.personalInformation)
   const { serviceHistory, mostRecentBranch } = useSelector<RootState, MilitaryServiceState>((state) => state.militaryService)
   const { ratingData } = useSelector<RootState, DisabilityRatingState>((state) => state.disabilityRating)
   const { data: userAuthorizedServices } = useAuthorizedServices()
+  const { data: personalInfo } = usePersonalInformation()
   const accessToMilitaryInfo = userAuthorizedServices?.militaryServiceHistory && serviceHistory.length > 0
   const theme = useTheme()
   const { t } = useTranslation(NAMESPACE.COMMON)
 
   const ratingPercent = ratingData?.combinedDisabilityRating
   const ratingIsDefined = ratingPercent !== undefined && ratingPercent !== null
   const combinedPercentText = ratingIsDefined ? t('disabilityRating.combinePercent', { combinedPercent: ratingPercent }) : undefined
+  const fullName = personalInfo?.fullName
```

Comment:
Since this is only used in one place, we probably don't need to store it in a variable, and can just use `personalInfo?.fullName` directly

Reply:
updated